man/make.conf.5

```diff
@@ -796,6 +796,14 @@ Have portage react strongly to conditions that may conflict with system
 security provisions (for example textrels, executable stack).  Read about
 the \fIQA_STRICT_*\fR variables in \fBmake.conf\fR(5).
 .TP
+.B suppress\-bindist\-binpkg
```

Comment:
Maybe add something about how it only does something if ACCEPT_RESTRICT excludes bindist? I know that's obvious, but it makes things clearer for users trying to understand how things fit together.